doc/introduction.rst

```diff
@@ -158,42 +177,45 @@ a command prompt::
 Testing your installation
 .........................
 
-To test if you have done everything right, open IPython and try the
-following, in the Python prompt::
+To verify whether everything is set up correctly, open IPython (an widely
+used enhanced version of the Python prompt, cf. below) and punch in the follwing
```

Comment:
missing 'o' in foll**o**wing

maybe 'type in' is simpler to understand than 'punch in'